src/routes/campaign/[slug]/+page.js

```diff
@@ -0,0 +1,13 @@
+export async function load ({params}) {
+    // if you want the data to be fetched before the page is shown
+    // (make the page slower to navigate to - user sees browser loading indicator)
+    // then use this approach:
+    
+    // return new Promise(resolve => {
+    //     setTimeout(() => {
+    //         resolve({value: `You traveled to slug: ${params.slug}`})
+    //     }, 400)
+    // })
+
+    return {slug: params.slug};
```

Comment:
Is `slug` a svelte keyword, or can you replace it with a more meaningful value like `campaignName`?

Reply:
Good call.  I changed it to `campaignKey`, since we'll need unique URLs even if we don't enforce unique campaign names